cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,158 @@ Cask Data Application Platform Release Notes
+  option on widget JSON in Hydrator plugins.
+
+- `CDAP-4836 <https://issues.cask.co/browse/CDAP-4836>`__ - Fixed a spacing issue for
+  metrics showing in Pipeline diagrams of the Hydrator UI.
+
+- `CDAP-4853 <https://issues.cask.co/browse/CDAP-4853>`__ - Fixed issues with the Hydrator
+  UI widgets for the Hydrator Kafka real-time source, JMS real-time source, and CloneRecord
+  transform.
+
+- `CDAP-4865 <https://issues.cask.co/browse/CDAP-4865>`__ - Enhanced the CDAP SDK to be
+  able to publish metadata updates to an external Kafka, identified by the configuration
+  property metadata.updates.kafka.broker.list. Publishing can be enabled by setting
+  metadata.updates.publish.enabled to true. Updates are published to the Kafka topic
+  identified by the property metadata.updates.kafka.topic.
+
+- `CDAP-4877 <https://issues.cask.co/browse/CDAP-4877>`__ - Fixed errors in Cask Hydrator
+  Plugins. Two plugin documents (core-plugins/docs/Database-batchsink.md and
+  core-plugins/docs/Database-batchsource.md) were removed, as the plugins have been moved
+  from core-plugins to database-plugins (to database-plugins/docs/Database-batchsink.md and
+  database-plugins/docs/Database-batchsource.md).
+
+- `CDAP-4889 <https://issues.cask.co/browse/CDAP-4889>`__ - Fixed an issue with upgrading
+  HBase tables while using the CDAP Upgrade Tool.
+
+- `CDAP-4894 <https://issues.cask.co/browse/CDAP-4894>`__ - Fixed an issue with CDAP
+  coprocessors that caused HBase tables to be disabled after upgrading the cluster to a
+  highly-available file system.
+
+- `CDAP-4906 <https://issues.cask.co/browse/CDAP-4906>`__ - Fixed the CDAP Upgrade Tool to
+  return a non-zero exit status upon error during upgrade.
+
+- `CDAP-4924 <https://issues.cask.co/browse/CDAP-4924>`__ - Fixed a PermGen memory leak
+  that occurred while deploying multiple applications with database plugins.
+
+- `CDAP-4927 <https://issues.cask.co/browse/CDAP-4927>`__ - Fixed the CDAP Explore
+  Service JDBC driver to do nothing instead of throwing an exception when a commit is
+  called. 
+  
+- `CDAP-4950 <https://issues.cask.co/browse/CDAP-4950>`__ - Added an ``'enableAutoCommit'``
+  property to the Cask Hydrator database plugins to enable the use of JDBC drivers that,
+  similar to the Hive JDBC driver, do not allow commits.
+
+- `CDAP-4951 <https://issues.cask.co/browse/CDAP-4951>`__ - Changed the upload timeout from the
+  CDAP CLI from 15 seconds to unlimited.
+
+Deprecated and Removed Features
+-------------------------------
+
+- See the :ref:`CDAP 3.3.1 Javadocs <javadocs>` for a list of deprecated and removed APIs.
+
+- The properties ``router.ssl.webapp.bind.port``, ``router.webapp.bind.port``,
+  ``router.webapp.enabled`` have been deprecated and will be removed in a future version.
+
+.. _known-issues-331:
+
+Known Issues
+------------
+- See the *Known Issues* of `version 3.2.0 <#known-issues-320>`_\ .
```

Comment:
No. There are no "3.3.0" Known Issues. The previous known-issues list is "3.2.0".

One problem is that there is no update of what is outstanding, except by checking each manually. But sending people to the issues database and all outstanding issues would probably not be a good idea.